src/node/txdownloadman.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <node/txdownloadman.h>
+#include <node/txdownloadman_impl.h>
+
+namespace node {
+
+TxDownloadManager::TxDownloadManager(const TxDownloadOptions& options) :
+    m_impl{std::make_unique<TxDownloadImpl>(options)}
+{}
+TxDownloadManager::~TxDownloadManager() = default;
+
+void TxDownloadManager::ActiveTipChange()
```

Comment:
To avoid the circular dependency, you could get rid of this file and move its contents to `txdownloadman_impl.cpp`.

Reply:
done